README.md

```diff
@@ -39,26 +34,44 @@ please install the redistributable installer for [Visual Studio 2013](https://ww
 
 ## Compiling from source
 
-Before building pc-ble-driver-py you will need to have Boost installed and some of its libraries statically compiled.
-To install and compile Boost, please follow the instructions here:
+Before building pc-ble-driver-py you will need make nrf-ble-driver available as a CMake module. The easiest way to do this is to install it with [vcpkg](https://github.com/Microsoft/vcpkg).
+
+    vcpkg install nrf-ble-driver:<triplet> --head
+
+*TODO: remove --head argument when port PR of nrf-ble-driver is merged with MSFT master*
+triplet must match the version of python you are building the binding for. To see the triplets supported, type:
```

Comment:
Could add a link to https://github.com/Microsoft/vcpkg/blob/master/docs/users/triplets.md for explanation of the term triplet.